src/com/layout/grid/selectable-grid.js

```diff
@@ -0,0 +1,31 @@
+import {h, Component} from 'preact/preact';
+import Grid from 'com/layout/grid/grid';
+import GridSelector from 'com/layout/grid/grid-selector';
+
+export default class SelectableGrid extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+			'columns': 3
+		};
```

Comment:
Indentation hard to folloe

Reply:
Yeah, it's minor, but it could be corrected.

Reply:
This one really confuses me. In atom it looks fine, must be a weird indentation with spaces and tabs thing.

Reply:
As a pythonite I growl every time I see tab in code. ;) probably a case of you having setting for how long a tab should be.